components/Wallet/Connect.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import { ConnectButton } from "@rainbow-me/rainbowkit";
+
+const ConnectWallet = () => {
+  // const connectWithMetamask = useMetamask();
+  // const disconnect = useDisconnect()
+  // const address = useAddress();
+
+  // useEffect(() => {
+  //   address !== undefined && toast.success('Successfully connected with MetaMask 🦊')
+  // }, [address])
```

Comment:
Is this necessary now? Add a comment if you want to do anything with this in the future.

Reply:
i forgot about this, ofc its not necessary, gj